data/weapons.txt

```diff
@@ -457,6 +457,32 @@ effect "proton impact"
 	"velocity scale" -.005
 
 
+outfit "Proton Turret"
+	category "Turrets"
+	cost 260000
+	thumbnail "outfit/proton turret"
+	"mass" 36
```

Comment:
I'd make it 32 tons. 36 (mass of turret) - 28 (mass of gun) = 8 additional mass on this turret in order to carry one gun. The average additional mass per gun for human turrets is 3.6, which rounds up to 4.

Reply:
I figured that the turret base itself was pretty heavy to support a weapon of this size. This is a fairly chunky Syndicate weapon, after all. Not that I wouldn't love 32-ton Proton Turrets, but that seems a little bit _too_ light. What do other folks think?

Reply:
Well the Plasma Turret at 60 tons and 25 tons per gun is using 10 tons to support both guns, which comes out to 5 tons each. The Proton Gun is only 3 tons heavier than the Plasma Cannon, so I'd say the Proton Turret should have at MOST 6 extra tons added on to it. But then we have the Electron Turret, which is 54 tons and 21 tons per gun, which means 12 tons of support/6 tons of support per gun. So clearly, the chunkiness of the gun isn't at play here, since the slim Electron Beam has more weight supporting it than the chunky Plasma Cannon. Since it's only a singular gun attached to a turret, I think that 4 tons is completely reasonable.

Reply:
@ the 8 minute ago edit
>Not that I wouldn't love 32-ton Proton Turrets, but that seems a little bit too light. I could maybe go for 34 or 35 though.

I can agree on a 34 ton turret.

Reply:
I tend to consider the extra mass of the turret as embodied by the base and the support structure, not on a per-gun basis. Evidence for this would be the Quad Blaster Turret vs the regular Blaster Turret: in each case, the base and mount contribute only five tons. Those same five tons support two guns or four. Also, we have several other single-large-weapon turrets that also have gun versions to use as a point of comparison:
- Sunbeam turret is 12 mass for the base and mount
- Moonbeam turret is 16 mass for the base and mount

That said, I think going down to 35 tons would be reasonable. 34 is... well, I want this PR to be merged. :slightly_smiling_face:

Reply:
I'd argue that the Sunbeam and Moonbeam are alien weapons so it's different than comparing to other human turrets. :)
And although the Moonbeam Turret is a single gun, the fire rate on that gun is increased when compared to the gun version of the Moonbeam.